### DISCUSSION TITLE
[Projects Beta] Hide completed Milestones automatically

### DISCUSSION THREAD

**Discussion by danielnordh** [community/community#10218] [Projects and Issues]
I can't find a way of hiding (automatically or manually) completed Milestones in the Project dashboard/overview.
This clutters up the dashboard pretty quickly when you want to keep open/current milestones and issues at the top.

**Comment by ghost** [comment]
I have the same issues

**Comment by luongvo** [comment]
+1

**Comment by dimapasko** [comment]
I am also experiencing the same issue. It would be helpful to learn about how other teams are resolving this.
